api/src/main/java/org/apache/cloudstack/api/response/RoleResponse.java

```diff
@@ -43,6 +43,10 @@
     @Param(description = "the description of the role")
     private String roleDescription;
 
+    @SerializedName(ApiConstants.IS_DEFAULT)
+    @Param(description = "true if role is default, false otherwise")
```

Comment:
nit - can you also explain what it means for a role to be default (default could be misleading, perhaps what you mean is if the role is built-in).

Reply:
default meaning this role(s) comes by default with cloudstack installation / upgrade. "default" used, in sync with similar usage at the offerings, accounts in CS.